.ddev/config.yaml

```diff
@@ -1,4 +1,3 @@
-APIVersion: v1.8.0
 name: introduction
```

Comment:
It's a good practice to ommit the name so DDEV will add it on the first start automatically from the folder name. This prevents naming conflicts. But I guess in this case you will have troubles afterwards because the rest of the config.

Reply:
yes, i think in this case it makes sense to keep the name